config/7.2.0/securebanking/ig/routes/routes-service/04-ob-as-authorize.json

```diff
@@ -36,7 +36,20 @@
             "type": "application/x-groovy",
             "file": "FapiCompliantAuthorizeRequestFilter.groovy"
           }
-        }        
+        },
+        {
+          "name": "ReSignIdTokenFilter",
+          "type": "ReSignIdTokenFilter",
+          "comment": "Re-sign the id_token returned by AM to fix OB keyId issue",
+          "config": {
+            "verificationSecretsProvider": "SecretsProvider-AmJWK",
+            "verificationSecretId": "any.value.in.regex.format",
```

Comment:
same config used when we verify that `access_token`s have been signed by AM as part of route security.